app/views/users/edit.html.erb

```diff
@@ -1,6 +1,27 @@
 <h1>Editing User</h1>
 
-<%= render 'form', user: @user %>
+
+
+ <%= form_for @user do |form| %>
+    <%= render partial: '/users/form', object: form %>
```

Comment:
we don't need /users here, the same directory is assumed if we just have form?
so, this would also be fine: 
`<%= render partial: 'form', object: form %>`